enhancement/genre_rules_manager.py

```
@@ -0,0 +1,528 @@
+            "",
+            "### 节奏要求",
+            f"- {self.rhythm.get('description', '')}",
+        ])
+        
+        if self.examples:
+            lines.extend([
+                "",
+                "### 写作示例"
+            ])
+            for i, ex in enumerate(self.examples, 1):
+                lines.append(f"  ❌ 差例{i}：{ex['bad']}")
```

Ruff (0.15.10)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)